.github/workflows/docker_all-modules.yml

```diff
@@ -18,6 +18,7 @@ on:
     # build and push all on tagged release
     tags:
       - "v*.*.*"
+      - "20[0-9][0-9]-[0-1][0-9]-[0-3][0-9]"
```

Comment:
I don't know if I agree with this. Can you tell me more about why you think we need this?

Reply:
I mostly just thought it might be nice to have periodically tagged docker images in addition to semantic, but I don't feel too strongly about it and think it is fine to remove this.

I also just realized that I'm not sure the current docker action would push these if they are not semantic, so I'm going to revert this change.